io.github.cybertimon.rapidraw.yml

```diff
@@ -0,0 +1,58 @@
+id: io.github.cybertimon.rapidraw
+
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+
+command: RapidRAW
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: binary
+    buildsystem: simple
+
+    sources:
+      - type: file
+        path: io.github.cybertimon.rapidraw.metainfo.xml
+      - type: file
+        url: https://github.com/CyberTimon/RapidRAW/releases/download/v1.3.10/03_RapidRAW_v1.3.10_ubuntu-24.04_amd64.deb
+        sha256: 54e9cb263b680267abbec612d5319dfd2cedd703b27c5dbc88fe80de22662afb
+        only-arches: [x86_64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/CyberTimon/RapidRAW/releases/latest
+          version-query: .tag_name
+          url-query: '"https://github.com/CyberTimon/RapidRAW/releases/download/" + $version + "/03_RapidRAW_" + $version + "_ubuntu-24.04_amd64.deb"'
+      - type: file
+        url: https://github.com/CyberTimon/RapidRAW/releases/download/v1.3.10/03_RapidRAW_v1.3.10_ubuntu-24.04-arm_arm64.deb
+        sha256: 8d087edc9c42815767ff77e3b72b5925daa5edee652f977d1b6b01a71df96419
+        only-arches: [aarch64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/CyberTimon/RapidRAW/releases/latest
+          version-query: .tag_name
+          url-query: '"https://github.com/CyberTimon/RapidRAW/releases/download/" + $version + "/03_RapidRAW_" + $version + "_ubuntu-24.04-arm_arm64.deb"'
+    build-commands:
+      - set -e
+
+      - mkdir deb-extract
+      - ar -x *.deb --output deb-extract
+      - tar -C deb-extract -xf deb-extract/data.tar.gz
+
+      - 'install -Dm755 deb-extract/usr/bin/RapidRAW /app/bin/RapidRAW'
+
+      - mkdir -p /app/lib/RapidRAW
+      - cp -r deb-extract/usr/lib/RapidRAW/. /app/lib/RapidRAW
+      - find /app/lib/RapidRAW -type f -exec chmod 644 {} \;
+
+      - sed -i 's/^Icon=.*/Icon=io.github.cybertimon.rapidraw/' deb-extract/usr/share/applications/RapidRAW.desktop
```

Comment:
Never use sed to change desktop files. It has no failure mode. Use the command `desktop-file-edit`.